src/main/java/mate/controller/IndexController.java

```diff
@@ -1,9 +1,32 @@
 package mate.controller;
 
+import java.io.IOException;
+import java.time.LocalDateTime;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import mate.dao.MyCoolResource;
 
 public class IndexController extends HttpServlet {
     private MyCoolResource myResource;
-    //TODO: implement
+
+    @Override
+    public void init(ServletConfig config) {
```

Comment:
```suggestion
    public void init() {
```
since we don't use config we can override the method with 0 input params